django_mongodb_engine/compiler.py

```diff
@@ -386,11 +389,15 @@ def insert(self, docs, return_id=False):
 
         collection = self.get_collection()
         options = self.connection.operation_flags.get('save', {})
+        if '_id' in doc:
+            collection.replace_one({'_id': doc['_id']}, doc, upsert=True, **options)
+            return_doc_id = doc['_id']
+        else:
+            result = collection.insert_one(doc, **options)
+            return_doc_id = result.inserted_id
```

Comment:
@ilavajuthy save() logic is changed now. Django-mongodb code always accepts save() with single document.

Reply:
looks good